fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -291,12 +291,14 @@ public static float[] ExtrapolateFromAboveAtIndexLocation(float[] stagesAtStruct
             }
             return extrapolatedStages;
         }
-        public float[] ExtrapolateFromBelowStagesAtIndexLocation(float[] WSEsAtLowest, float interval, int i)
+        public static float[] ExtrapolateFromBelowStagesAtIndexLocation(float[] WSEsAtLowest, float interval, int i, int numInterpolatedStagesToCompute)
```

Comment:
because we went with static

Reply:
make sure this is commented, so we know we're using access modifiers and static specifically for testing. Because someone eventually (probably future us) might wonder why the hell we did that if we don't.